src/AKCore/Scripts/VueComponents/Countdown/CountDownApp.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <div class="countdown-container" v-if="timeLeft">
+    <div class="countdown">
+      <div class="countdown-item">
+        <span class="countdown-number">{{ days }}</span>
+        <span class="countdown-label">Days</span>
+      </div>
+      <div class="countdown-item">
+        <span class="countdown-number">{{ hours }}</span>
+        <span class="countdown-label">Hours</span>
+      </div>
+      <div class="countdown-item">
+        <span class="countdown-number">{{ minutes }}</span>
+        <span class="countdown-label">Minutes</span>
+      </div>
+      <div class="countdown-item">
+        <span class="countdown-number">{{ seconds }}</span>
+        <span class="countdown-label">Seconds</span>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, ref, computed, onMounted, onBeforeUnmount } from "vue";
+
+//@ts-ignore
+// eslint-disable-next-line no-undef
+const date = (): any => selectedDate;
```

Comment:
Om syftet här är att komma åt en global variabel så använd samma sätt som gjorts här:

https://github.com/LudHag/AKCore/blob/765c109bf4bd17ea90d83d0d9f1d0edc3d7ec171/src/AKCore/Scripts/vueapps.ts#L39

Dvs definiera med declare const ute i vueapps samt skicka in som typad variabel i komponenten.